lib/ansible/modules/windows/win_robocopy.py

```diff
@@ -82,6 +84,12 @@
     src: C:\DirectoryOne
     dest: C:\DirectoryTwo
     flags: /E /PURGE /XD SOME_DIR /XF SOME_FILE /MT:32
+
+- name: Sync one file from a remote UNC path in recursive and purging mode, specifying additional special flags
+  win_robocopy:
+    src: "\\\\Server1\\Directory One\\"
```

Comment:
Are you able to change this to `src: \\Server1\Directory One\`, when not using double quotes you don't need to double up on the backslash.

Reply:
Can you see if that format works?  We see this format breaking (could be a bug).  Ends up putting everything in the cmd (including flags)

dest param shows in the output, but errors with:

"msg": "ERROR : No Destination Directory Specified."



    "cmd": " \\\\srcserver\\share\\subfolder\\Sub Folder\\Subfolder\\ C:\\Installs blah.zip /Z /R:1 /W:30 /NP",

Reply:
@aaronk1 sorry I didn't reply sooner, I've never tried win_robocopy but `\\Server1\Directory One\` is seem as the exact same chars in Ansible as `"\\\\Server1\\Directory One\\"` so there shouldn't be any behaviour differences between them. It sounds like robocopy isn't quoting them properly as the cmd being returned doesn't have the quotes at all.

Reply:
@jborean93 No worries.  I've been busy too.  Will get back to this ASAP.